CoffeeChat/Cells/AboutPearTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  AboutPearTableViewCell.swift
+//  CoffeeChat
+//
+//  Created by Amy Chin Siu Huang on 10/11/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class AboutPearTableViewCell: UITableViewCell {
+
+    private let paragraphLabel = UILabel()
+    
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        selectionStyle = .none
+        
+        contentView.backgroundColor = .clear
+        backgroundColor = .clear
+        paragraphLabel.numberOfLines = 0
+        paragraphLabel.backgroundColor = .clear
+        contentView.addSubview(paragraphLabel)
+        
+        setupConstraints()
+    }
+    
+    private func setupConstraints() {
+        paragraphLabel.snp.makeConstraints { make in
+            make.leading.trailing.equalToSuperview().inset(36)
+            make.centerY.equalTo(contentView.snp.centerY)
+        }
+    }
+    
+    func configure(for aboutParagraph: AboutParagraph) {
+        let boldText = aboutParagraph.bold
```

Comment:
I left this comment on Phillip's PR last time too but this is a simpler / cleaner way you can combine multiple styles together. I'm giving an example of what I have done for another project. I think Phillip also did something like this (he already made an extension) so you can refer to that. 

Basically you add an extension like this with styles.

```
extension NSMutableAttributedString {
     func boldFont14(_ value:String) -> NSMutableAttributedString {
         let attributes: [NSAttributedString.Key : UIFont] = [
         .font : .boldSystemFont(ofSize: 14)
         ]
         self.append(NSAttributedString(string: value, attributes:attributes))
         return self
     }

      func normalFont14(_ value:String) -> NSMutableAttributedString {
         let attributes: [NSAttributedString.Key : UIFont] = [
             .font : .systemFont(ofSize: 14),
         ]
         self.append(NSAttributedString(string: value, attributes:attributes))
         return self
     }
 }
```
Then later when you have to make the text. you do

```
private func setupFriendAcceptCell(fromUser: String) {
         notificationLabel.attributedText =
             NSMutableAttributedString()
                 .boldFont14(fromUser)
                 .normalFont14(" accepted your friend request ")
         layoutNotificationLabel()
     }
```